AK/Optional.h

```diff
@@ -36,6 +36,8 @@ namespace AK {
 template<typename T>
 class alignas(T) [[nodiscard]] Optional {
 public:
+    using value_type = T;
```

Comment:
Please use `TitleCase` for type names.

Reply:
gotcha, wasn't sure of the casing on this one since I just copied the alias from `Vector`